low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -152,5 +228,5 @@ def get_realstor_encl_data(fru: str):
 
 # if __name__ == "__main__":
 #     storage = StorageMap()
-#     health_data = storage.get_health_info(rpath="nodes[0]>storage[0]>hw>controllers")
+#     health_data = storage.get_health_info(rpath="nodes[0]>storage[0]")
```

Comment:
Is this checked for below?
rpath="nodes[0]>storage[0]>fw>disk_groups"
rpath="nodes[0]>storage[0]>hw>logical_volumes"

Reply:
Yes. working as expected